spring-integration-core/src/main/java/org/springframework/integration/support/management/BaseChannelMetrics.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.support.management;
+
+import org.springframework.jmx.export.annotation.ManagedMetric;
+import org.springframework.jmx.support.MetricType;
+
+/**
+ * @author Gary Russell
```

Comment:
Hmmm - I wonder why checkstyle didn't catch this.

Reply:
We just don't that rule enabled:
```
<!-- 		<module name="JavadocStyle"> -->
		<!-- 			<property name="checkEmptyJavadoc" value="true"/> -->
		<!-- 		</module> -->
```

There are really a lot of our classes without JavaDocs.
Although it may treat even this `@author` presence as not empty...

Reply:
Ah, right; (no, I have seen checkstyle report a missing JD, even with `@author, @since` in projects with this rule enabled).